Bookish/Views/Member/DeleteMember.cshtml

```diff
@@ -0,0 +1,17 @@
+@model Bookish.Models.MemberModel
+@{
+    ViewData["Title"] = "Delete Member";
+}
+
+<div class="text-center">
+    @if (Model != null)
+    {
+        <div class="alert alert-success" role="alert">
+            Successfully deleted member from the library! :)
+        </div>
+    }
+    
+    <div class="container">
+        <h1>Random text to make page work</h1>

```

Comment:
I'm curious about this... does the page not work if this text isn't there?

Reply:
No, I was testing something early on and forgot that page still existed!